client/src/templates/Introduction/components/block-header.tsx

```diff
@@ -45,6 +48,9 @@ function BlockHeader({
           </span>
           <DropDown />
         </span>
+        {blockIntroArr && blockIntroArr.length && (
```

Comment:
```suggestion
        {!isEmpty(blockIntroArr) && (
```
Nitpicky, I know, but this is more declarative. (btw, we need to get `isEmpty` from `lodash-es` for reasons of tree shaking).